test/WebJobs.Script.Tests/Rpc/RpcInitializationServiceTests.cs

```diff
@@ -38,51 +37,60 @@ public RpcInitializationServiceTests()
                                              .Returns(Task.CompletedTask);
         }
 
-        [Fact]
+        [Fact(Skip = "Investigate")]
```

Comment:
Recommend referencing issue in the skip statement.

Likely you need this test suite to implement IDisposable and dispose your services between each test.